archethic-foundation/archethic-node
# Store utxo on genesis pool

# Description

Fixes #1241 

## Type of change

- New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Unit tests
Manual tests:
- receive few inputs (UCO) from the faucet, 
- analyse the ets table via the observer, to see the list of unspent inputs (3)
- make some transaction
- analyse the ets table via the observer, to see the list of unspent inputs (-consumed inputs)

# Checklist:

- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- My changes generate no new warnings
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
- Any dependent changes have been merged and published in downstream modules
